File: 백준/Bronze/5532. 방학 숙제/방학 숙제.py
l = int(input())
a = int(input())
b = int(input())
c = int(input())
d = int(input())

result = 0

while(True):
    if a <= 0 and b <= 0:
        break
    
    a -= c
    b -= d
    
    result += 1

print(l - result)

File: 백준/Bronze/5532. 방학 숙제/방학 숙제.c
#include <stdio.h>

int main(void) { 
   int l, a, b, c, d;
   scanf("%d", &l);
   scanf("%d", &a);
   scanf("%d", &b);
   scanf("%d", &c);
   scanf("%d", &d);

   int result = 0;
   while(1){
      if(a <= 0 && b <= 0)   break;
      a -= c;
      b -= d;

      result++;
   }

   printf("%d", l - result);
   
   return 0;
}